at_secondary/at_secondary_server/test/notify_list_verb_test.dart

```diff
@@ -365,6 +365,49 @@ void main() {
     });
     tearDown(() async => await tearDownFunc());
   });
+
+  group('A group of test to verify notify fetch', () {
```

Comment:
Please add a test for non-existent notification ID

Reply:
Added test for non existent notification